src/dotnet/commands/dotnet-add/dotnet-add-package/LocalizableStrings.cs

```diff
@@ -13,6 +13,8 @@ internal class LocalizableStrings
 
         public const string SpecifyExactlyOnePackageReference = "Please specify one package reference to add.";
 
+        public const string CmdPackageDescription = "Name of the package to be added.";
```

Comment:
You're killing me. Why does fixing this parsing issue require more strings. Can you paste old and new help output?